### DISCUSSION TITLE
Example usage with Astro?

### DISCUSSION THREAD

**Discussion by digoburigo** [gnat/css-scope-inline#15] [Q&A]
Hello. I would like to know if its possible to use this with the [Astro](https://astro.build/) framework?

**Comment by AceroM** [comment]
Using the `is:inline` directive (https://docs.astro.build/en/reference/directives-reference/#isinline) should work after importing the script. 

example:
```
<div>
  <style is:inline>
    me {
      background-color: red;
    }
  </style>
...
</div>
```

**Comment by AceroM** [comment]
One more change you need to do is adding this in your `<head>` so that you can avoid FOUC

```
    <script
      is:inline
      src="https://cdn.jsdelivr.net/gh/gnat/css-scope-inline/script.js"
    ></script>
```

the `is:inline` property is important to opt out of astro's script processing. I can make a PR to add to the README?

**Comment by gnat** [comment]
Gonna convert this to a discussion as it may be an ongoing thing. Kudos @AceroM for your insight as a fellow Astro user :+1: Could add a very short note about it in the Technical FAQ- but I'd generally like to avoid adding content that applies to only one framework.
